index.js

```diff
@@ -260,12 +304,16 @@ function findBestSubtitleFile(list, locales = []) {
     .sort((x, y) => y.priority - x.priority)[0];
 }
 
+function isSilentVideo(audio_bitrate) {
+  return audio_bitrate === 0 || (audio_bitrate && audio_bitrate <= 10);
```

Comment:
```suggestion
  return audio_bitrate <= 10;
```

Reply:
copy + pasta, I'm fairly certain you need the audio_bitrate but just incase it's null or undefiend but will remove the === 0 part since it's redundant.